src/FsLex.Core/fslexast.fs

```diff
@@ -1,21 +1,30 @@
 (* (c) Microsoft Corporation 2005-2008.  *)
 
-module internal FsLexYacc.FsLex.AST
+module FsLexYacc.FsLex.AST
 
 open System.Collections.Generic
 open System.Globalization
-open Internal.Utilities.Text.Lexing
+open FSharp.Text.Lexing
```

Comment:
Why the namespace change here?

Reply:
I believe it's because we're not internalizing the FsLexYacc.Runtime anymore, and instead making it a normal project reference